src/content/app/tools/blast/components/listed-blast-submission/ListedBlastSubmission.tsx

```diff
@@ -162,11 +167,12 @@ const StatusElement = ({ jobs }: { jobs: BlastJob[] }) => {
   if (hasRunningJobs) {
     return <span className={elementClasses}>Running...</span>;
   } else if (hasFailedJobs) {
-    if (jobs.length === 1 || jobs.every((job) => job.status === 'FAILURE')) {
-      return <span className={elementClasses}>Failed</span>;
-    } else {
-      return <span className={elementClasses}>Some jobs failed</span>;
-    }
+    return (
+      <div>
+        <span className={elementClasses}> Job failed </span>
+        <QuestionButton helpText={<FailedSubmissionHelpText />} />
+      </div>
```

Comment:
You probably want to adjust the vertical alignment here:

![image](https://user-images.githubusercontent.com/6834224/208388360-25262f5f-19f3-4096-8dc9-62a909096573.png)